roboteam_ai/src/conditions/ShotAtGoal.cpp

```diff
@@ -0,0 +1,32 @@
+//
+// Created by robzelluf on 3/14/19.
+//
+
+#include "ShotAtGoal.h"
+
+namespace rtt{
+namespace ai {
+
+ShotAtGoal::ShotAtGoal(std::string name, bt::Blackboard::Ptr blackboard)
```

Comment:
The name of this condition is a little bit weird. a shot at which goal? Does it return succes when a shot at goal happens or when it is possible for an attacker (and an attacker from which team then?)

Reply:
Renamed